### ISSUE TITLE
同一个封装的View，不同的位置，骨架动画出现错乱。

### ISSUE THREAD

**Issue by mancongiOS** [tigerAndBull/TABAnimated#189]
### New Issue Checklist

* [x] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
![image](https://user-images.githubusercontent.com/16421332/144035585-5bf05162-3a57-455a-a47b-07d692364795.png)

        let allWidth = UIDevice.bt.width - 22
        let registerWidth = allWidth * 0.392
        let managerWidth = (allWidth - registerWidth) / 2
        
        addSubview(registerLabel)
        registerLabel.snp.makeConstraints {
            $0.center.equalTo(self)
            $0.height.equalTo(40)
            $0.width.equalTo(registerWidth)
        }
        
        addSubview(managerLabel)
        managerLabel.snp.makeConstraints {
            $0.centerY.equalTo(self)
            $0.height.equalTo(40)
            $0.width.equalTo(managerWidth)
            $0.left.equalTo(10)
        }

        addSubview(setUpDateLabel)
        setUpDateLabel.snp.makeConstraints {
            $0.centerY.equalTo(self)
            $0.height.equalTo(40)
            $0.width.equalTo(managerWidth)
            $0.right.equalTo(-10)
        }

**Comment by mancongiOS** [comment]
想只显示BDTopAndBottomView的骨架，不显示其内部的元素。 
是通过这样的方式么？ 
                manager.animation()?(12)?.layer.removeAllSublayers()
//                manager.animation()?(9)?.layer.loadStyle = .remove

这样做不生效。

**Comment by tigerAndBull** [comment]
tabAnimated.withoutSubViewsClass = BDTopAndBottomView.class
